elastic/integrations
# Prometheus Input Package

- Enhancement

## What does this PR do?

The Prometheus Input Package is developed in this PR.

## Checklist



## How to test this PR locally

Bring up any service pushing metrics to Prometheus Endpoint. Add the Prometheus Input Integration from Kibana. Add the appropriate input configurations. The metrics should be received in Kibana. 

## Related issues

- Closes https://github.com/elastic/integrations/issues/4615

## Screenshots

<img width="1697" alt="Screenshot 2023-05-11 at 10 19 08 PM" src="https://github.com/elastic/integrations/assets/102962586/1bb66411-16f9-472b-a607-dbee118fd684">
